test/simplifyExpression/basicsSearch/mixedNumber.test.js

```diff
@@ -0,0 +1,103 @@
+const convertMixedNumberToImproperFraction = require(
+  '../../../lib/simplifyExpression/basicsSearch/convertMixedNumberToImproperFraction');
+const mixedNumber = require('../../../lib/simplifyExpression/basicsSearch/mixedNumber');
+const TestUtil = require('../../TestUtil');
+
+function testIsMixedNumber(input, output) {
+  TestUtil.testBooleanFunction(mixedNumber.isMixedNumber, input, output);
+}
+
+function testIsNegativeMixedNumber(input, output) {
+  TestUtil.testBooleanFunction(mixedNumber.isNegativeMixedNumber, input, output);
+}
+
+function testGetWholeNumber(input, output) {
+  input = TestUtil.flattenAndParse(input);
+  TestUtil.testFunctionOutput(mixedNumber.getWholeNumber, input, output);
+}
+
+function testGetNumerator(input, output) {
+  input = TestUtil.flattenAndParse(input);
+  TestUtil.testFunctionOutput(mixedNumber.getNumerator, input, output);
+}
+
+function testGetDenominator(input, output) {
+  input = TestUtil.flattenAndParse(input);
+  TestUtil.testFunctionOutput(mixedNumber.getDenominator, input, output);
+}
+
+function testConvertMixedNumberToImproperFraction(exprString, outputList, outputStr) {
+  TestUtil.testSubsteps(convertMixedNumberToImproperFraction, exprString, outputList, outputStr);
+}
+
+describe('isMixedNumber', function () {
+  const tests = [
+    ['5(1)/(6)', true],
+    ['19(2)/(3)', true],
+    ['-1(7)/(8)', true],
+    ['4*(1/2)', false],
+    ['(1/2)3', false],
+    ['3*10/15', false],
```

Comment:
oh okay I have no idea what's going on then haha - I'll just go by what you're saying is passed to the app, though that's so weeirrrd :p 

Yeah mathparser will just parse and it'll work :p whatever we want we can probably have it support that, that's the beauty of a custom parser!